diff --git a/date.theme b/date.theme
index 5cec8b4..4634b7b 100755
--- a/date.theme
+++ b/date.theme
@@ -349,7 +349,7 @@ function theme_date_display_range($variables) {
   }
 
   // Wrap the result with the attributes.
-  $output = '<div class="date-display-range">' . t('!start-date to !end-date', array(
+  $output = '<div class="date-display-range">' . t('!start-date lidz !end-date', array(
     '!start-date' => $start_date,
     '!end-date' => $end_date,
   )) . '</div>';
